rules.adoc

```diff
@@ -17,7 +17,7 @@ endif::basebackend-html[]
 :icons: font
 :numbered:
 
-These are the official Soccer Simulation rules for *the RoboCupJunior Simulation Demo Competition in Feb 2021*.
+These are the official Soccer Simulation rules for *the RoboCupJunior Simulation 2021*.
```

Comment:
I think you should say: 
These are the official Soccer Simulation rules for *the RoboCupJunior 2021*

Reply:
Fixed :-)